ext/symengine/symengine.c

```diff
@@ -46,4 +47,10 @@ void Init_symengine() {
     c_rational = rb_define_class_under(m_symengine, "Rational", c_basic);
     rb_define_alloc_func(c_rational, cbasic_alloc);
     rb_define_method(c_rational, "initialize", crational_init, 1);
+
+    //Constants
+    c_constants = rb_define_class_under(m_symengine, "Constants", c_basic);
+    rb_define_const(m_symengine, "PI", cconstants_pi());
+    rb_define_const(m_symengine, "E", cconstants_e());
+    rb_define_const(m_symengine, "EulerGamma", cconstants_euler_gamma());
```

Comment:
I would have suggested to stay with the style used in SymEngine but it is a convention in Ruby to name your constants in all caps. See the convention used for constants [here](http://itsignals.cascadia.com.au/?p=7).